src/libstore/build/derivation-goal.cc

```diff
@@ -765,64 +765,6 @@ Goal::Co DerivationGoal::tryLocalBuild() {
 }
 
 
-static void chmod_(const Path & path, mode_t mode)
```

Comment:
This was dead code before hand, right? Maybe put it in a different commit, because it's a bit confusing why it disappear now.